### PR TITLE
Pass coverage config file by absolute path

### PR DESCRIPTION
**Pull request by takluyver**
The kernel may not be running in the same working directory as the test process.

### CODE REVIEW

**Comment by takluyver** [comment]
@vidartf would you have a few minutes to look at this? The actual change I wanted to make is just two lines in `cover.py` - the other changes were things needed to get the tests working.

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ install:
   - pip install six
   # Travis needs to update pip/setuptools
   - pip install --upgrade setuptools pip
-  - pip install .
+  - pip install -e .
```
Comment (vidartf) [inline .travis.yml]: Did this end up being needed in the end? I'd prefer to keep it a non-dev install if possible.

Comment (takluyver) [reply .travis.yml]: No, I'm not sure it is. I'll try reverting that and see if it goes OK.

**Comment by takluyver** [comment]
You were right, the dev install wasn't needed.

**Comment by takluyver** [comment]
Thanks! I'll make a new release soon, because this was something that came up in our use of nbval. Anything else you'd like to get in before I do that?

**Comment by vidartf** [comment]
I can't think of anything I need for a release, so feel free to do it as you please!

**Comment by vidartf** [comment]
Hmm. I found this commit though: https://github.com/computationalmodelling/nbval/commit/80979c9e9cbab30dbc80f8307c447801904d950e Maybe that can be replaced with `nbsphinx>=0.3.1`?

**Comment by vidartf** [comment]
But that is only for docs, so not critical!

**Comment by takluyver** [comment]
Thanks, done in a4db7dba723cf78426b72f5f7ec8a81768cd1ed9

**Pull request merged**